tests/test_gpsr.py

```diff
@@ -78,7 +78,7 @@ def circuit_sequence(n_qubits: int) -> QuantumCircuit:
     ],
 )
 @pytest.mark.parametrize("ops_op", [pyq.Z, pyq.X, pyq.Y])
-@pytest.mark.parametrize("dtype", [torch.complex64, torch.complex128])
+@pytest.mark.parametrize("dtype", [torch.complex128])
```

Comment:
Not sure it is useful for a single parameter.

Reply:
torch.complex64 will be added later, after the tensor modifications brought by #220 are in

Reply:
It gives an error due to missing the dtype argument when creating identities, but we will fix directly after the MR.